packages/flutter/test/cupertino/tab_scaffold_test.dart

```diff
@@ -47,6 +47,10 @@ void main() {
     selectedTabs = <int>[];
   });
 
+  tearDownAll(() {
+    imageCache.clear();
+  });
```

Comment:
Per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#make-each-test-entirely-self-contained it's generally better to put this in each test, that way you can easily copy tests to other files and generally easily tell what exactly each test is doing.

Reply:
Clean up for each test will improve test isolation.
Clean up just at the end will improve test performance.

I guess one of the reasons image cashing exists is exactly performance of testing.

But I am open for both ways.